package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stargate",
-  "version": "0.0.0",
+  "version": "0.3.0",
```

Comment:
I wouldn't manually bump this.

Instead, run:
`npm version major` to increment the major version
`npm version minor` to increment the minor version
`npm version patch` to increment the patch version.

This will create an increment commit for you. 🙂 

If you prefer to do it manually, I'd run `npm install` after so you can commit the updated package-lock.json too. Otherwise `npm ci` will fail.

Reply:
Oh good call. The GitHub Action updates this though 🤔

Reply:
Ahh that's unfortunate. 🤔